api/src/main/java/org/pmiops/workbench/api/DisksController.java

```diff
@@ -20,15 +21,18 @@
 public class DisksController implements DisksApiDelegate {
   private static final Logger log = Logger.getLogger(DisksController.class.getName());
 
+  private final DiskService diskService;
```

Comment:
I couldn't get this to work when I tried it, but I wish we could just annotate these and then not have to add the boilerplate to the constructor.